cluster/manifests/coredns-local/daemonset-coredns.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
     application: coredns
     # application: kubernetes # step 2
     component: coredns
-    version: v1.8.4
+    version: v1.8.7
```

Comment:
this is not part of the message. Do you want to change the message and description to make this through?
Happy to have this too within this PR, because of e2e runtime

Reply:
Weird, I somehow included this in a rebase, that was a mistake, thanks for noticing it!

It was already rolled out here: https://github.com/zalando-incubator/kubernetes-on-aws/pull/4941 :shrug: